perllib/FixMyStreet/Cobrand/Brent.pm

```diff
@@ -1345,9 +1344,20 @@ sub bulky_can_refund { 0 }
 sub bulky_free_collection_available { 0 }
 sub bulky_hide_later_dates { 1 }
 
+=head2 bulky_allowed_property
+
+Brent allows bulky waste collection for any property
+with a domestic refuse collection so check against
+any domestic refuse service id
+
+=cut
+
 sub bulky_allowed_property {
-    my ( $self, $property ) = @_;
-    return $self->bulky_enabled;
+    my ($self, $id) = @_;
+
+    return $self->bulky_enabled
+            && $id
+            && grep {$id eq $_} qw/262/;
```

Comment:
I guess this was in case there was more than one in future? For now, I assume this could be `return $self->bulky_enabled && $id == 262;` :) (it'll always be set, I assume)